python/dgl/_ffi/object.py

```diff
@@ -52,21 +52,6 @@ def __reduce__(self):
         cls = type(self)
         return (_new_object, (cls, ), self.__getstate__())
 
-    def __getstate__(self):
-        # TODO(minjie): TVM assumes that a Node (Object in DGL) can be serialized
-        #   to json. However, this is not true in DGL because DGL Object is meant
-        #   for runtime API, so it could contain binary data such as NDArray.
-        #   If this feature is required, please raise a RFC to DGL issue.
-        raise RuntimeError("__getstate__ is not supported for object type")
-
-    def __setstate__(self, state):
```

Comment:
Why remove this? The default pickle for C++ object might be only holding the handle address, which might cause other bugs.

Reply:
reverted